tests/test_other.py

```diff
@@ -10316,7 +10316,7 @@ def test_split_module(self, customLoader):
     self.emcc_args += ['-g', '-Wno-experimental']
     self.emcc_args += ['--post-js', test_file('other/test_split_module.post.js')]
     if customLoader:
-      self.emcc_args += ['--post-js', test_file('other/test_load_split_module.post.js')]
+      self.emcc_args += ['--pre-js', test_file('other/test_load_split_module.post.js')]
```

Comment:
I guess the file needs to be rename too now.    I would just call it `test_split_module.pre.js` to match the other filenames.